docs/app/pages/components/components-sections/tree-view/tree-view-custom-node/tree-view-custom-node.component.less

```diff
@@ -1,5 +1,8 @@
 .tree-view-example-container {
     height: 250px;
+    .spinner {
+        margin: 6px auto;
```

Comment:
Instead of this, perhaps adding top/bottom margin to the mini activity spinner by default so that it has the same 10px height as the previous version? That should reduce the need for custom tweaks like this.